From: Boyuan Yang <byang@debian.org>
Date: Fri, 3 Sep 2021 02:17:43 -0400
Subject: configure.ac: Do not manually set localedir

Manually setting localedir conflicts with the automatically
generated localedir variable in autoconf 2.70+.

Bug-Debian: https://bugs.debian.org/978823
---
 configure.ac | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 754676f..fdd856d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -13,8 +13,6 @@ IT_PROG_INTLTOOL([0.40.0])
 
 GETTEXT_PACKAGE="${PACKAGE}"
 AC_SUBST(GETTEXT_PACKAGE)
-localedir='$(prefix)/$(DATADIRNAME)/locale'
-AC_SUBST(localedir)
 
 # Workaround to make aclocal get the right flags
 AC_SUBST(ACLOCAL_AMFLAGS, "\${ACLOCAL_FLAGS}")
